cli/cli.go

```diff
@@ -11,6 +11,11 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
+Version Change Summary:
+1.1.0 - 09/28/2021 - Replaced jwt-go with golang-jwt/jwt module.  This
+                     requires a 'go get github.com/golang-jwt/jwt` to
```

Comment:
Note: we should not have to do a 'go get' since we use Go modules (Go will get the module for us automatically)

Did you run "go mod tidy" to make sure we were updating just the required set of modules?